### DISCUSSION TITLE
Command for reverting changes in 'destination directory' (from the source directory.)

### DISCUSSION THREAD

**Discussion by notDavid** [twpayne/chezmoi#3144] [Q&A] [answered]
Hi there, just getting started so i have a noob question;

1. I directly change a file in the 'destination directory', let's say `echo "test" >> ~/.bashrc`
2. Now I want to revert/restore these changes (so basically i would `copy ~/.local/share/chezmoi/dot_bashrc ~/.bashrc`

Is there a chezmoi command for this? (I expected something like `chezmoi restore ~/.bashrc`.)

Thank you!

**Comment by twpayne** [comment, marked as answer]
`chezmoi apply ~/.bashrc`

You can use `chezmoi diff ~/.bashrc` to see what changes would be made by `chezmoi apply`.

**Reply by notDavid** [reply]
Ah of course, i knew it would be easy but... it's a lot to take in when starting :-)
Thank you for the help!
